python/core/auto_generated/qgsfileutils.sip.in

```diff
@@ -84,6 +84,22 @@ and "/home/user/projects" exists but no "2018" subfolder exists, then the functi
 
 .. versionadded:: 3.2
 %End
+
+    static QStringList findFile( const QString &fileName, const QString &basepath, int maxDepth = 4, int driveMargin = 3, const QString &currentDir = "" );
+%Docstring
+Will check ``basepath`` in an outward spiral up to ``maxDepth`` levels to check if ``fileName`` exists.
+
+:param fileName: Name of the file to find
+:param basepath: current basepath of the file
+:param maxDepth: limit the number of time the search can move up
+:param driveMargin: limits how close the search can get to the drive level
```

Comment:
A "drive"? I don't think it makes much sense for a *nix user.
So, first question is: is this a windows specific implementation?
If it is not (as I hope) can you rename this parameter and explain in the documentation of the argument what it exactly does?

Reply:
If you are willing to provide better terminology I can certainly change it. I was not too sure how to express this variable, but the gist is to maintain a buffer between the places that will be searched and the root. This is to prevent the alg from searching the full pc/server and contain it in a sub branch of the architecture.

I can call it root margin but I doubt this would help either.

Reply:
Please see the changes that I made, they should make things clearer, thank you for pushing me to change them!